lib/supavisor.ex

```diff
@@ -160,6 +160,34 @@ defmodule Supavisor do
     end)
   end
 
+  @spec del_all_cache(String.t()) :: [map()]
+  def del_all_cache(tenant) do
+    Logger.info("Deleting all cache for tenant #{tenant}")
+    {:ok, keys} = Cachex.keys(Supavisor.Cache)
```

Comment:
I've had issues dumping all the keys of a cache frequently. This may not happen frequently and it may not be a terribly large key space. Mentioning just in case.

You could instead use a Cachex.Query to filter the keys for a tenant.

[Here's a query example](https://github.com/Logflare/logflare/blob/22a79f98a0d5cd6dd439310d53a5565f61497a25/lib/logflare/context_cache.ex#L80) I did with Logflare recently.

Reply:
ohh, cool

Reply:
I think it should be fine with only `Cachex.stream!()`